app/models/like.rb

```diff
@@ -1,4 +1,11 @@
 class Like < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+  after_create :update_post_likes_counter
```

Comment:
- [ ] Kindly use the `after_save` callback here instead as we are trying to call the update method upon saving a like to the database